decidim-dev/app/models/decidim/dev/coauthorable_dummy_resource.rb

```diff
@@ -1,8 +1,10 @@
 # frozen_string_literal: true
 
 module Decidim
-  module DummyResources
+  module Dev
     class CoauthorableDummyResource < ApplicationRecord
+      self.table_name = "decidim_dummy_resources_coauthorable_dummy_resources"
```

Comment:
Are these necessary to define manually? Why they can't be created automatically as before but with `dev` (i.e.  `decidim_dev_coauthorable_dummy_resources`)?